python/test/scripts/example.py

```diff
@@ -76,6 +76,7 @@ def main(args):
             environment_id = environment_id,
             event_type = "SDK Event",
             layer = "Compute",
+            source = "Example.py",
```

Comment:
Can we change the example to look more like the source is a system or product?  In our case, events are coming from DataKeeper (a product/tool that can generate events) or WSFC, etc. but the example looks like the source is a file or script.